README.md

```diff
@@ -3,12 +3,12 @@
 This Repo is meant to be for all the assignments and projects that are done by me and my friends while following a timetable similar to the one suggested by Siraj in this [video](https://www.youtube.com/watch?v=-OvRVlqKebI) and this [repo](https://github.com/llSourcell/Learn_Computer_Science_in_5_Months).
 
 Week wise sources we followed are mentioned below.
-Our Mini-Projects and Assignements can be found in the individual week folders.
+Our Mini-Projects and Assignments can be found in the individual week folders.
 
 # Time-Table and Sources
 
 ##### Week 0 - Git,Github and Markdown
 
 ##### Week 1 - Python
 
-The project is in progress and will be updated weekly.
+The projects are in progress and will be updated weekly.
```

Comment:
let this be is i am referring to the whole project here

Reply:
Ok fine, just change Assignment spelling then as proposed. Silly but necessary.

Reply:
yep agree